### PR TITLE
Hw02 express

### PR DESCRIPTION
**Pull request by Bob-UA**

### CODE REVIEW

File: schemas/contacts.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const Joi = require("joi");
+
+const addSchema = Joi.object({
+  name: Joi.string().required(),
```
Comment (ceREAL133) [inline schemas/contacts.js]: add message to required

Comment (Bob-UA) [reply schemas/contacts.js]: Hi.
I checked "Joi" documentation, there is no information about message field in ".required( )". 
By "schema" testing it is working, I have message "{"message": "\"name\" is required"}" when I imitate put request without "name".
Can you please write where exactly I should add message?

Comment (ceREAL133) [reply schemas/contacts.js]: name: Joi.string()
    .messages({
      'string.empty': `"name" cannot be an empty field`,
    })
    
    you could do something like that

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: schemas/contacts.js (hunk 1 above)
Comment (Bob-UA) [reply schemas/contacts.js]: > name: Joi.string() .messages({ 'string.empty': `"name" cannot be an empty field`, })
> 
> ```
> you could do something like that
> ```

Changed and tested, everything works.

"const addSchema = Joi.object({
  name: Joi.string()
    .required()
    .messages({ "any.required": 'missing required name field' }),
  email: Joi.string()
    .required()
    .messages({ "any.required": 'missing required email field' }),
  phone: Joi.string()
    .required()
    .messages({ "any.required": 'missing required phone field' }),
});"

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**